test/integration-v2.sh

```diff
@@ -48,6 +48,7 @@ done
     --listen localhost:9103 \
     --listen-internal localhost:9104 \
     --forward-url=http://localhost:9105/api/v1/receive \
+    --memcached=localhost:11211 \
```

Comment:
how did you add this line? this is already in the file: https://github.com/openshift/telemeter/blob/master/test/integration-v2.sh#L51

Are you working on the latest of the master branch?

Reply:
ah nvm, you removed it and added it back